model/verseInfo.js

```diff
@@ -0,0 +1,130 @@
+const { parseHttpResponse } = require("selenium-webdriver/http");
+const utils = require("./utils");
+
```

Comment:
We are returning a new type of object (which contains arrays of other objects). Please add a definition for this schema object and call it something like VerseInformation